backend/dataall/modules/datasets/services/share_notification_service.py

```diff
@@ -0,0 +1,101 @@
+from dataall.db import models
+from dataall.db.api import Notification
+from dataall.modules.datasets.db.models import Dataset
+
+
+class ShareNotificationService:
```

Comment:
As this class only has static methods, shouldn't these functions be on module level? (without a class)

I'd also rename the file to follow the pattern in the folder: dataset_ ...

Reply:
I don't think that we will gain more benefits by removing the class. Yeah, it doesn't carry any functional role here, but it groups the code with the same functionality.

About the renaming: yes, I completely agree with you. This class will be changed when the sharing module comes up :)